### DISCUSSION TITLE
Translations: stakeholder review and issue tracker.

### DISCUSSION THREAD

**Discussion by hammerdirt-analyst** [hammerdirt-analyst/IQAASL-End-0f-Sampling-2021#34] [General]
The place to start correcting mistakes is here. Drop the requested changes in the discussion. Once they are reviewed, an issue will be opened and the changes will be reflected once the issue is closed.

See the first comment for a good method on how to signal a correction.

**Comment by hammerdirt-analyst** [comment]
Corrections for the foreword and executive report

**Foreword – Using this report**
From this: 
In practical terms there was a better chance of finding plastic construction waste on the beach in Bönigen and Unterseen than the other municipalities. However, the chances of finding industrial sheeting were approximately equal anywhere but the most might be found at at Brienze (67p/100m). 

To this:
In practical terms there was a better chance of finding plastic construction waste on the beach in Bönigen and Unterseen than the other municipalities. However, the chances of finding industrial sheeting were approximately equal anywhere but the most might be found at Brienz (67p/100m)

**Executive summary**

From this:
Figure 4: Brienze | Ziegelbrüucke

To this:
Figure 4: Brienz | Ziegelbrücke 

_These map dates are incoherent:_
This report is a summary and analysis of the litter surveys conducted and the methods employed in Switzerland from March 2020 through August 2021. → Why does the caption of the map say “Map of survey locations March 2020 - July 2021. ”? And also in chapter “Lakes and rivers” in the Executive Summary you mention “The lakes and rivers were sampled from 2020-03 through 2021-05”.

_From this:_
Industrial pellets and expanded foams < 5mm both occurred in significant quantities but identified in less than 50% of the surveys (median of 0), indicating high counts at specific locations. While both are micro -plastics, their use, origin and rate of occurrence are different depending on the survey area region. Industrial pellets are raw materials used in injection molding processes and whereas foamed plastic beads are the result of fragmentation of expanded polystyrene. For more information on Llocation, quantities and rate of individual object occurrence see [Lakes and rivers](https://hammerdirt-analyst.github.io/IQAASL-End-0f-Sampling-2021/lakes_rivers.html#allsurveys).

_To this:_
Industrial pellets and expanded foams < 5mm both occurred in significant quantities but identified in less than 50% of the surveys (median of 0), indicating high counts at specific locations. While both are micro plastics, their use, origin and rate of occurrence are different depending on the survey area region. Industrial pellets are raw materials used in injection molding processes whereas foamed plastic beads are the result of fragmentation of expanded polystyrene. For more information on location, quantities and rate of individual object occurrence see [Lakes and rivers](https://hammerdirt-analyst.github.io/IQAASL-End-0f-Sampling-2021/lakes_rivers.html#allsurveys).

Question:
For Alps survey methodology and results see [Les Alpes](https://hammerdirt-analyst.github.io/IQAASL-End-0f-Sampling-2021/alpes_valaisannes.html#lesalpes). → Isn’t the correct title “The Alps and the Jura”?

_From this:_
For communicating pollution quantities, converting results into a simple metric of average pieces per 100m is useful as the average is generally larger and rarely 0. However, the average may be twice the median when extreme values are considered causing confusion regarding the difference between observed and reported results. Communicating the range of likely values or the likelihood of finding an object is more informative and repeatable when following similar protocols. For example, when interpretating the quantities of industrial pellets identified on Lac Léman: 

There were 1’387 plastic production pellets (GPI) or 5% of all objects identified on Lac Léman. The number of pellets per 100m ranges from 0 -1,033 depending on the region. For the lake in general there is a approximately a 40% chance of finding at least one pellet at a survey, anywhere. In some locations, like Genève (60% chance) or Préverenges (80% chance), production pellets are regular features on these beaches, and amounts between 3p/100m and 56p/100m are common.

_To this:__
For communicating pollution quantities, converting results into a simple metric of average pieces per 100m is useful as the average is generally larger and rarely 0. However, the average may be twice the median when extreme values are considered causing confusion regarding the difference between observed and reported results. Communicating the range of likely values or the likelihood of finding an object is more informative and repeatable when following similar protocols. For example, when interpretating the quantities of industrial pellets identified on Lac Léman: 

There were 1’387 plastic production pellets (GPI) or 5% of all objects identified on Lac Léman. The number of pellets per 100m ranges from 0 -1,033 depending on the region. For the lake in general there is approximately a 40% chance of finding at least one pellet at a survey, anywhere. In some locations, like Genève (60% chance) or Préverenges (80% chance), production pellets are regular features on these beaches and amounts between 3p/100m and 56p/100m are common.

_From this:__
On a national scale, the survey results were stable with respect to 2017. However, there was a general decrease in the quantity of food, drink and tobacco objects. Infrastructure objects and fragmented plastics and foams did not decline and some locations may have experienced sharp increases. Pandemic restrictions limiting large outdoor gatherings may have had a beneficial effect on the reduction of food, drink and tobacco items. The greatest increases in infrastructure related objects were in Valais, Vaud and Brienze, which are locations in proximity to the Rhône and Aare rRivers discharge points.

_To this:_
On a national scale, the survey results were stable with respect to 2017. However, there was a general decrease in the quantity of food, drink and tobacco objects. Infrastructure objects and fragmented plastics and foams did not decline and some locations may have experienced sharp increases. Pandemic restrictions limiting large outdoor gatherings may have had a beneficial effect on the reduction of food, drink and tobacco items. The greatest increases in infrastructure related objects were in Valais, Vaud and Brienz, which are locations near the Rhône and Aare rivers discharge points.

**Reply by hammerdirt-analyst** [reply]
These corrections were handled with issue #36 Translations and syntax corrections

**Reply by hammerdirt-analyst** [reply]
### Survey Area comments

The comments submitted by user sam and amanda have been integrated in the template. There are two samples... one based on the Aare survey area and Ticino survey area.

see issue Stakeholder review comments 1: #37

**Reply by hammerdirt-analyst** [reply]
this issue closed with 5259a3be97e0e6bc506b6aada7ac458fd92b1383

**Comment by hammerdirt-fieldops** [comment]
chapter 3.1. Erhebungsorte und Landnutzungsmerkmale
change Für den Zeitraum zwischen 03.2020 und 05.2021,
**Für den Zeitraum zwischen März 2020 und Mai 2021**

**Reply by hammerdirt-analyst** [reply]
What is to be done with this?

**Comment by hammerdirt-analyst** [comment]
Terms and corrections from the FOEN-translation of the Foreword and Executive Summary, which need to respected in the whole report. 

Foreword 

1. Der Medianwert (50. Perzentil) der Datenerhebungen Ergebnisse wird als die Anzahl der Objekte pro 100m (p/100m) der Küstenlinie angegeben.  
2. Please add spacing after using a unit: 
3. Der Medianwert (50. Perzentil) der Erhebungsergebnisse gibt die Anzahl Gegenstände pro 100 m (p/100 m) Küstenlinie an.  Datenerhebungen Ergebnisse or Ergebnisse der Datenerhebungen = please translate with «Erhebungsergebnisse» (survey results) 
4. ERGEBNISSE DER ERHEBUNG (title of chapter 1) = please translate with «Erhebungsergebnisse» (survey results) 
5. Basisanalyse = please translate with «Analyse der Erhebungsergebnisse» (baseline analysis) 
6. Metrische Bewertung = please translate with «Bewertungsparameter» (assessment metric)) 
7. Code-Gruppen = please translate with «Codegruppen» (for instance title chapter 16) 
8. Das Profil der Landnutzung = please translate with «Das Landnutzungsprofil» (land use profile) (for instance title chapter 17) 
9. Quatres Cantons = please translate with «Vierwaldstättersee» 
10. Die am häufigsten gefundenen Objekte = please translate with “Die am häufigsten gefundenen Gegenstände” (The most common objects) 
11. Seen und Flüsse = please translate with «Seen und Fliessgewässer» (Lakes and rivers) (for instance title chapter 1) 
12. Umfrageergebnisse = please translate with “Erhebungsergebnisse” (survey results) 

Codes (consistency of translation in the Executive summary with code groups in chapter 16) 

ShapeGeneral remark: In the German version of chapter 16 there is a different order in each code group compared to the English version. In the English version you have two columns of numbers on the left, which is not the case for the German version. 

The code group tobacco (in German «Tabak») needs to be translated into «Tabakwaren» (for instance in chapter 16.2) 

 Code G95 Cotton bud/swab sticks | In German “Wattestäbchen”, please translate with “Wattestäbchen / Tupfer” 

Here an example from chapter 16.2.1:

**Reply by hammerdirt-analyst** [reply]
This remains to be done: Quatres Cantons = please translate with «Vierwaldstättersee»
the rest was covered with [i42 new german terms and corrections](https://github.com/hammerdirt-analyst/IQAASL-End-0f-Sampling-2021/commit/8e164da203687aa0497f7f059488898ea4e6abc2)

**Comment by hammerdirt-fieldops** [comment]
The Alps and the Jura chapter
2.2. Survey area cumulative totals
Below the map change text _Between_ 2020-03 and 2021-09 to correct date 2021-05 and 2021-08

**Comment by hammerdirt-fieldops** [comment]
The exact number different G codes used
